contracts/vesting/Genesis.sol

```diff
@@ -150,11 +150,11 @@ contract GenesisClaim is Ownable {
      * @param _address address
      * @return uint256
      */
-    function redeemableFor(address _address) public view returns (uint256) {
-        Term memory info = terms[_address];
-        uint256 max = circulatingSupply().mul(info.percent).mul(1e3);
+    function redeemableFor( address _address ) public view returns (uint) {
```

Comment:
Why remove the explicit `uint256`

Reply:
Left out by accident. Did not not notice removed in the commit.